.github/workflows/main.yml

```
@@ -0,0 +1,48 @@
+name: gpt-review
+
+on:
+  pull_request:
+    types: ["opened", "synchronize"]
+    branches:
+      - main
+
+jobs:
+  gpt-review:
+    name: gpt-review
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    env:
+      GH_TOKEN: ${{ github.token }}
+    steps:
```

*セキュリティ* 観点の *low* レベルの指摘
 このコードはGH_TOKENを環境変数として設定していますが、github.tokenを直接使用することも可能です。独自のトークンを使用すると、トークンの権限を細かく制御できますが、誤って公開してしまうリスクもあります。

```suggestion
独自のトークンを使用する場合は、トークンが漏れないように注意してください。また、可能であればgithub.tokenを直接使用することをご検討ください。
```